uwrt_mars_rover_hw/config/rover_controllers.yaml

```diff
@@ -5,7 +5,7 @@ uwrt_mars_rover:
     publish_rate: 50
 
   drivetrain_velocity_controller:
-    type: diff_drive_controller/DiffDriveController
+    type: uwrt_diff_drive_controller/UWRTLoopDiffDriveController
```

Comment:
I dont think this change is supposed to be here?

Reply:
Yeah, I changed it back to diff_drive_controller/DiffDriveController